Automattic/wp-calypso
# Fediverse: remove support for Jetpack/Atomic sites

Our support in Marketing Connections for Jetpack/Atomic sites has been a bit hacky. It recently grew more fragile to the point where it makes the most sense to rip it out. **Business users can simply install the ActivityPub plugin**, this was just guided them through that process, albeit not well enough to not be confusing.

Fixes #96700

## Proposed Changes

* Remove Fediverse from the Connections page for Jetpack/Atomic sites. They can use the ActivityPub plugin directly.

## Why are these changes being made?
* With the move in #96605 to a new design, there has been some additional strange behavior: clicking the section open would close it immediately. A little more debugging showed that the plugins list is currently pretty unreliable, which is what prompted the weird behavior in #96700.

## Testing Instructions

* With this PR active, navigate to Marketing Connections on an Atomic site. There should be no "Fediverse" section.
* For a Simple site, navigate to Marketing Connections. There **should be** a Fediverse section.

<img width="1219" alt="Screenshot 2024-12-02 at 15 23 53" src="https://github.com/user-attachments/assets/d462bc5a-4c56-401b-8562-9e99beeef6dd">


## Pre-merge Checklist
